distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -1633,624 +1859,788 @@ type Column
+       ## Examples
+       ### Get the first element of a column.
 
+       ```
              import Standard.Examples
 
              example_first = Examples.integer_column.first
+       ```
     first self -> Any ! Index_Out_Of_Bounds =
         self.implementation.first self
 
-    ## GROUP Standard.Base.Selections
-       ICON select_row
+    ## ---
+       group: Standard.Base.Selections
+       icon: select_row
+       ---
        Returns the last element in the column, if it exists.
-
        If the column is empty, this method will return a dataflow error
        containing an `Index_Out_Of_Bounds`.
 
-       > Example
-         Get the last element of a column.
+       ## Examples
+       ### Get the last element of a column.
 
+       ```
              import Standard.Examples
 
              example_last = Examples.integer_column.last
+       ```
     last self -> Any ! Index_Out_Of_Bounds =
         self.implementation.last self
 
-    ## GROUP Standard.Base.Selections
-       ICON order
+    ## ---
+       group: Standard.Base.Selections
+       icon: order
+       ---
        Returns a column containing the values of `self` column with their order
        reversed.
 
-       > Example
-         Reverse the values of a column to produce a new column.
+       ## Examples
+       ### Reverse the values of a column to produce a new column.
 
+       ```
              import Standard.Examples
 
              example_reverse = Examples.integer_column.reverse
+       ```
     reverse self -> Column | Any =
         self.implementation.reverse self
 
-    ## GROUP Standard.Base.Metadata
-       ICON metadata
+    ## ---
+       group: Standard.Base.Metadata
+       icon: metadata
+       ---
        Returns a column of numbers, in which every entry denotes how many times
        the value at the given position occured before.
 
-       > Example
-         Count duplicate occurences of values in a column.
+       ## Examples
+       ### Count duplicate occurences of values in a column.
 
+       ```
              import Standard.Examples
 
              example_duplicate_count = Examples.integer_column.duplicate_count
+       ```
     duplicate_count self -> Column | Any =
         self.implementation.duplicate_count self
 
-    ## PRIVATE
-       Provides a simplified text representation for display in the REPL and errors.
+    ## ---
+       private: true
+       ---
+       Provides a simplified text representation for display in the REPL and
+       errors.
     to_text self -> Text =
         self.implementation.to_text_implementation self
 
-    ## PRIVATE
+    ## ---
+       private: true
+       ---
     pretty self -> Text =
         self.implementation.pretty_implementation self
 
-    ## ALIAS count, kurtosis, maximum, mean, median, minimum, mode, skew, standard_deviation, statistic, sum, variance
-       GROUP Standard.Base.Statistics
-       ICON transform4
+    ## ---
+       aliases:
```

Comment:
I thought the syntax was [count, kurtosis, ....]
Does this work too?

Reply:
Yes both are valid yaml array formats.